src/utils/propertyUtils.js

```diff
@@ -0,0 +1,13 @@
+export function getPropertyValue(object, propertyName) {
+    const propertyDescriptor = Object.getOwnPropertyDescriptor(object, propertyName);
+    if (propertyDescriptor.get) {
+        try {
+            return propertyDescriptor.get()
+        }
+        catch {
+            return propertyDescriptor.get
```

Comment:
Is this right?

Reply:
this would return the getter function rather then the value?

Reply:
Yes, instead of crashing when getting a property we will return a "getter" description here.